### ISSUE TITLE
`package 'liburiparser' not found` upon `collada_dom/create_dom.sh`

### ISSUE THREAD

**Issue by 130s** [tork-a/88_tools#8]
On freshly installed Ubuntu Trusty, I got this:

```
curl -sL https://raw.githubusercontent.com/tork-a/88_tools/master/collada_dom/create_dom.sh | bash
:
-- found boost version: 105400
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.1") 
-- libxml2 found
-- checking for module 'minizip'
--   package 'minizip' not found
-- compiling minizip from sources and linking statically
-- checking for module 'liburiparser'
--   package 'liburiparser' not found
CMake Error at /usr/share/cmake-2.8/Modules/FindPkgConfig.cmake:283 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-2.8/Modules/FindPkgConfig.cmake:337 (_pkg_check_modules_internal)
  CMakeLists.txt:252 (pkg_check_modules)


-- Configuring incomplete, errors occurred!
See also "/tmp/tmp48158/collada-dom2.4-dp-2.4.3.0.orig/build_dir/CMakeFiles/CMakeOutput.log".
make: *** [build] Error 1
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```

`ros-indigo-rtmros-nextage` is already installed.

After `apt-get install liburiparser-dev`, the dpkg build started to progress. Is this the right workaround?

**Comment by k-okada** [comment]
Yes

## 

◉ Kei Okada

2015/07/23 9:04、Isaac I.Y. Saito notifications@github.com のメッセージ:

**Comment by 130s** [comment]
Closed via #9
